nix/checks.nix

```diff
@@ -10,6 +10,30 @@ _: {
           reuse lint
           touch $out
         '';
+        pycodestyle =
+          pkgs.runCommandLocal "pycodestyle" { nativeBuildInputs = [ pkgs.python3.pkgs.pycodestyle ]; }
+            ''
+              cd ${../.}
```

Comment:
nitpick but `${self}` can be used to refer to the flake root in more portable way than relative path

Reply:
Fixed in https://github.com/tiiuae/ghaf-infra/pull/304/commits/fda27a0da3c970c229892cfbc7159e7aaa21cd99